tests/test_cart.py

```diff
@@ -153,9 +153,9 @@ def test_cant_buy_any_tickets_if_fares_are_not_available(db, user_client):
 @mark.parametrize(
     "ticket_category, expected_ticket_types",
     [
-        ('personal', ['TRSP', 'TRLP', 'TRTP', 'TRCP', 'TRDP']),
-        ('company', ['TRSC', 'TRLC', 'TRTC', 'TRCC', 'TRDC']),
-        ('student', ['TRSS', 'TRLS', 'TRTS', 'TRCS', 'TRDS']),
+        ('personal', ['TRSP', 'TRLP', 'TRTP', 'TRCP', 'TRDP', 'TRPP']),
```

Comment:
why TRPP ? and not TRPT? I am just unable to remember the meaning of each combination...

Reply:
Marc added the sprint category, represented by the code P. So RPP is Regular Sprint Personal. Not sure why P is sprint though :-)

Reply:
TRPP = regular, sprint, personal

TRPT does not exist, since we don't have a T group.